cmd/minikube/cmd/start.go

```diff
@@ -747,7 +747,7 @@ func validateDriver(ds registry.DriverState, existing *config.ClusterConfig) {
 	}
 
 	r := reason.MatchKnownIssue(reason.Kind{}, st.Error, runtime.GOOS)
-	if r.ID != "" {
+	if r != nil && r.ID != "" {
 		exitIfNotForced(*r, st.Error.Error())
```

Comment:
how about move the logic inside the exitIfNotForced func, that way it should still Exit but without the Match ?

Reply:
that way minikube will still exit if it is nil

Reply:
here the idea was that it should use the 'reason' if it is already set and call exitIfNotForced with it, otherwise 'compile' the reason and then call the exitIfNotForced:

https://github.com/kubernetes/minikube/blob/a05f887651bd65102c6559f3c30439af3e792427/cmd/minikube/cmd/start.go#L749-L782

this pr just makes sure that before checking if reason ID is set, first check if r is not nil